### PR TITLE
Update live templates to assume null-safety

### PR DESCRIPTION
**Pull request by JuYeong0413**
Update IntelliJ live templates to null safety as VS Code snippets are updated in https://github.com/Dart-Code/Dart-Code/commit/2d2f7e222a43392717d6fd064e7c936b517a21b4.

* Add `?` operator to the `Key key` argument
* Update `inh` to match the [docs](https://api.flutter.dev/flutter/widgets/InheritedWidget-class.html)
* Add `late` keyword to the `AnimationController` in `stanim`

Examples:

https://user-images.githubusercontent.com/33684401/118861513-8ca5e200-b917-11eb-9713-f35fc50e0352.mov

https://user-images.githubusercontent.com/33684401/118864419-eeb41680-b91a-11eb-9eb4-7da7aa8bcb71.mov

https://user-images.githubusercontent.com/33684401/118861491-87489780-b917-11eb-95d9-f24e07ef5924.mov

/cc @stevemessick @AlexV525

### CODE REVIEW

**Review by stevemessick — APPROVED** [review]
Thanks @JuYeong0413 ! The demo movies are a nice touch.

File: resources/liveTemplates/stanim.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  const $NAME$({Key key}) : super(key: key);
+  const $NAME$({Key? key}) : super(key: key);
 
   @override
   _$NAME$State createState() => _$NAME$State();
 }
 
 class _$NAME$State extends State<$NAME$> with SingleTickerProviderStateMixin {
-  AnimationController _controller;
+  late AnimationController _controller;
```
Comment (AlexV525) [inline resources/liveTemplates/stanim.txt:9]: I'd more prefer this:
```suggestion
  late final AnimationController _controller = AnimationController(vsync: this);
```

**Review by AlexV525 — COMMENTED** [review]
Missed the review window before it gets merged.

File: resources/liveTemplates/stanim.txt (hunk 1 above)
Comment (JuYeong0413) [reply resources/liveTemplates/stanim.txt:9]: Oh yeah, I agree. I'll open another pull request for this :)

**Pull request merged**